lib/authenticate.tsx

```diff
@@ -16,6 +17,21 @@ export async function getUser(req) {
   return req ? req.user : null;
 }
 
+export async function getProfile(req) {
+  const rawProfileData = await fetch("http://localhost:3000/api/profile", {
+    headers: {
+      Cookie: req.headers.cookie
```

Comment:
need to include cookie from request for SSR. otherwise, `/api/profile` will 403

Reply:
```javascript
const rawProfileData = await fetch("http://localhost:3000/api/profile", {
    headers: req.headers,
});
```
Not sure what else is going down in the headers, but we could just do this? For some reason I thought the headers were already populated

Reply:
they do for client-side routing but this is for server-side rendering where the server itself is doing the `fetch` call, which is why we need to grab the `req.headers` from the client `req` 

good call tho, will do that instead

Reply:
Ah gotcha, kk